File: aAlvarezAssignment3/GasolineCar.py
from Car import Car
class GasolineCar(Car):
    def __init__(self,makeAndModel="none assigned",numberOfDoors=4,maximumNumberOfPassengers=5,gasTankSize=-1):
        super().__init__(makeAndModel,numberOfDoors,maximumNumberOfPassengers)
        self.tank= gasTankSize

    def getGasTankSize(self):
        return self.tank
    
    def setGasTankSize(self,size):
        self.tank =size

    def __str__(self):
        return super().__str__() + "\nGas Tank Size: " + str(self.tank)

File: aAlvarezAssignment4/GasolineCar.java
package aAlvarezAssignment4;

public class GasolineCar extends Car{
    private double gasTankSize;
    GasolineCar(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors,double gasTankSize){
        super(makeAndModel,maximumNumberOfPassengers,numberOfDoors);
        this.gasTankSize= gasTankSize;
}

    GasolineCar(){
        super();
        this.gasTankSize= -1;
    }

    public void setGasTankSize(double g){
        this.gasTankSize= g;
    }

    public double getGasTankSize(){
        return gasTankSize;
    }

    public String toString(){
        return (super.toString() + "\nGas Tank Size: " + gasTankSize);
    }
}

